app/src/main/java/org/p2p/wallet/auth/widget/HintEditText.kt

```diff
@@ -1,85 +1,29 @@
-/*
- * This is the source code of Telegram for Android v. 5.x.x.
- * It is licensed under GNU GPL v. 2 or later.
- * You should have received a copy of the license in this archive (see LICENSE).
- *
- * Copyright Nikolai Kudashov, 2013-2018.
- */
 package org.p2p.wallet.auth.widget
 
+import androidx.appcompat.widget.AppCompatEditText
 import android.content.Context
-import android.content.res.Resources
-import android.graphics.Canvas
-import android.graphics.Color
-import android.graphics.Paint
-import android.graphics.Rect
 import android.util.AttributeSet
 import android.view.KeyEvent
-import androidx.appcompat.widget.AppCompatEditText
 
 class HintEditText : AppCompatEditText {
+
     private var hintText: String? = null
-    private var textOffset = 0f
-    private var spaceSize = 0f
-    private var numberSize = 0f
-    private val paint = Paint()
-    private val rect = Rect()
 
     var onEmptyDelete: (() -> Unit)? = null
 
-    constructor(context: Context?) : super(context!!) {
-        paint.color = Color.LTGRAY
-    }
-
-    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs) {
-        paint.color = Color.LTGRAY
-    }
-
-    constructor(context: Context?, attrs: AttributeSet?, defStyleAttrs: Int) : super(
-        context!!, attrs, defStyleAttrs
-    ) {
-        paint.color = Color.LTGRAY
-    }
+    constructor(context: Context) : super(context)
```

Comment:
better with @JvmOverloads as we always do

Reply:
It's not working, the text style is being dropped. 
I didn't have time to debug and decided to revert back as it now.

But first, I made as we always do.

Let's skip this one for now, have no time for investigation